src/model/AirLoopHVAC_Impl.hpp

```diff
@@ -78,17 +78,21 @@ class MODEL_API AirLoopHVAC_Impl : public Loop_Impl {
 
   void autosizeDesignSupplyAirFlowRate();
 
-  Node supplyInletNode() const override;
+  double designReturnAirFlowFractionofSupplyAirFlow() const;
 
-  Node supplyOutletNode() const override;
+  bool setDesignReturnAirFlowFractionofSupplyAirFlow(double designReturnAirFlowFractionofSupplyAirFlow);
 
-  std::vector<Node> supplyOutletNodes() const override;
+  virtual Node supplyInletNode() const override;
```

Comment:
I added `virtual` anywhere `override` was used n this file. Because that's what it is. Serves no purposes really but it's clearer and more correct.